repo_dev.py

```diff
@@ -1,7 +1,7 @@
 """
 Simple shim for running the repo program during development.
 """
-import ga4gh.cli
+import ga4gh.cli.repomanager as cli_repomanager
```

Comment:
On the other hand (coupled with your other suggestion above), this would create circular references